### PR TITLE
Patch rubocop

### LINKED ISSUE #172: rake spec has any offenses detected
```ruby
$ bundle exec rake spec
...

cocoapods-keys/.rubocop_cocoapods.yml: Style/MultilineOperationIndentation has the wrong namespace - should be Layout
Inspecting 22 files
....CC.....CC..CC.CC..

Offenses:

lib/keyring_liberator.rb:66:30: C: Prefer using YAML.safe_load over YAML.load.
      Keyring.from_hash(YAML.load(path.read)) if path.file?
                             ^^^^
lib/name_whisperer.rb:28:25: C: Omit parentheses for ternary conditions.
        error_message = (xcodeprojects.length > 1) ? 'found too many' : "couldn't find any"
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/pod/command/keys/list.rb:33:22: C: Omit parentheses for ternary conditions.
            prefix = (index == data.length - 1) ? ' └ ' : ' ├ '
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/pod/command/keys/rm.rb:84:13: C: Redundant return detected.
            return keys.select { |e| e =~ create_regex(@key_name) }
            ^^^^^^
lib/pod/command/keys/rm.rb:86:13: C: Redundant return detected.
            return keys.select { |e| e == @key_name }
            ^^^^^^
spec/functional_spec.rb:5:1: C: Block has too many lines. [66/25]
describe 'CocoaPodsKeys functional tests' do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/functional_spec.rb:57:1: C: Use 2 spaces for indentation in a heredoc by using some library(e.g. ActiveSupport's String#strip_heredoc).
pod keys set KeyWithData "such-data" TestProject ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/key_master_spec.rb:8:1: C: Block has too many lines. [37/25]
describe CocoaPodsKeys::KeyMaster do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/keyring_spec.rb:6:1: C: Block has too many lines. [36/25]
describe KeyringLiberator do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:22:1: C: Block has too many lines. [146/25]
describe CocoaPodsKeys, '#plugin' do ...
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:47:5: C: %w-literals should be delimited by [ and ].
    %w(target targets).each do |target_tag|
    ^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:68:3: C: Block has too many lines. [44/25]
  context 'with a single target defined in the Podfile' do ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:87:5: C: Block has too many lines. [28/25]
    %w(target targets).each do |target_tag| ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:87:5: C: %w-literals should be delimited by [ and ].
    %w(target targets).each do |target_tag|
    ^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:126:3: C: Block has too many lines. [63/25]
  context 'with two targets defined in the Podfile' do ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:146:5: C: Block has too many lines. [46/25]
    %w(target targets).each do |target_tag| ...
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/plugin_spec.rb:146:5: C: %w-literals should be delimited by [ and ].
    %w(target targets).each do |target_tag|
    ^^^^^^^^^^^^^^^^^^

22 files inspected, 17 offenses detected
rake aborted!
Command failed with status (1): [bundle exec rubocop lib spec Rakefile...]
```

### PR DESCRIPTION
**Pull request by ykws**
fixes #172

### CODE REVIEW

**Comment by orta** [comment]
Thanks @ykws - we really need to keep ruby 2.0 support around - any chance you can add `activesupport < 5` to the Gemfile?

**Comment by ykws** [comment]
@orta Thank you for the review. I tried it.
Success `bundle exec rake spec`

File: lib/keyring_liberator.rb [hunk 1]
```diff
@@ -63,7 +63,7 @@ def self.get_all_keyrings
     end
 
     def self.get_keyring_at_path(path)
-      Keyring.from_hash(YAML.load(path.read)) if path.file?
+      Keyring.from_hash(YAML.safe_load(path.read)) if path.file?
```
Comment (orta) [inline lib/keyring_liberator.rb]: any chance you can check for `YAML.safe_load`'s existence before using it? in the CI it doesn't seem to exist (could be a ruby 2.0 thing )

Comment (ykws) [inline lib/keyring_liberator.rb]: sorry. my local ruby was ​​2.4 . I changed it to ruby ​​2.0 and retry `bundle exec rake spec` and succeeded.

**Comment by orta** [comment]
Amazing! Thanks!

**Pull request merged**